draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -198,14 +195,11 @@ which is important for low-latency applications that do not require
 guaranteed delivery services, such as Audio/Video streaming.
 
 In addition to the integration into DCCP services, implementers or future specification could choose MP-DCCP for other use cases like
-3GPP 5G multi-access solutions (e.g., Access Traffic Steering, Switching, and Splitting (ATSSS) specified in {{TS23.501}}) or hybrid access networks
-{{I-D.lhwxz-hybrid-access-network-architecture}}{{I-D.muley-network-based-bonding-hybrid-access}}, in
-which MP-DCCP can be used for load balancing, seamless session handover and bandwidth aggregation
-using a framework such as {{I-D.amend-tsvwg-multipath-framework-mpdccp}}.
+3GPP 5G multi-access solutions (e.g., Access Traffic Steering, Switching, and Splitting (ATSSS) specified in {{TS23.501}}) or hybrid access networks which either combine a 3GPP and a non-3GPP access or a fixed and cellular access between user user-equipment/residential gateway and operator network. MP-DCCP can be used in these scenarios for load balancing, seamless session handover and bandwidth aggregation when non-DCCP traffic like IP, UDP or TCP is encapsulated into MP-DCCP.
```

Comment:
/which/that/?

Reply:
https://github.com/tsvwg/ietf-multipath-dccp/pull/307/commits/1533bd679543010e314bcb772d6e35711bc4b922